build-tests/install-test-workspace/workspace/typescript-oldest-test/package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "heft-oldest-compiler-test",
+  "name": "typescript-oldest-test",
   "description": "Building this project tests Heft with the oldest supported TypeScript compiler version",
   "version": "1.0.0",
   "private": true,
   "main": "lib/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "heft build --clean"
+    "build": "heft clean --clear-cache && heft build"
```

Comment:
Should we just add `--clear-cache` to `heft build`/`heft test`

Reply:
`--clear-cache` should be the default IMO, because that cache is not reliable.